src/vehicle_type.h

```diff
@@ -80,4 +80,14 @@ enum EngineImageType : uint8_t {
 	EIT_PREVIEW    = 0x21,  ///< Vehicle drawn in preview window, news, ...
 };
 
+/** Randomisation triggers for vehicles */
+enum class VehicleRandomTrigger : uint8_t {
+	NewCargo, //!< Affected vehicle only: Vehicle is loaded with cargo, after it was empty.
+	Depot, //!< Front vehicle only: Consist arrived in depot.
+	Empty, //!< Front vehicle only: Entire consist is empty.
+	AnyNewCargo, //!< All vehicles in consist: Any vehicle in the consist received new cargo.
+	Callback32, //!< All vehicles in consist: 32 day callback requested rerandomisation
```

Comment:
Isn't `///<` the common way in OpenTTD's code to add these doxygen comments?

Reply:
Yes, it is.